test/contains.js

```diff
@@ -28,6 +28,36 @@ describe('contains', function() {
     assert.strictEqual(R.contains(new Just([42]), [new Just([42])]), true);
   });
 
+  it('dispatches to `indexOf` method', function() {
```

Comment:
is this name mismatch a first? it strikes me as odd in this case. The dispatching behavior simply falls out of the implementation. But `contains` returns a `Boolean` while `indexOf` returns a `Number`. It's not a match.